_docs/conversions.md

```diff
@@ -0,0 +1,180 @@
+```
+
+Note that the client one is a shaded jar.
+
+### Web Service Prerequisites
+
+* GitHub token - Learn how to create tokens on GitHub here. You will need the scope "repo".
+
+* GitHub organization(s) - Your GitHub token must have access to at least a single existing GitHub organization. The organization can be changed as long as the the GitHub token still has access to it. The Write API web service currently does not create GitHub organizations. The name of this organization must match the Quay.io organization. This organization will contain the repository that will be created.
+
+* Quay.io organization and Quay.io token
+
+    You will need an existing Quay.io organization. The Write API currently does not create Quay.io organizations. The name of this organization must match the GitHub organization. Once you have an organization, a token must be created for it.
+
+    This organization will contain the repository that will be created. Changing the Quay.io organization requires the token to be recreated/changed too.
+
+    Learn how to create a token on Quay.io for your organizations here under the heading "Generating a Token (for internal application use)". You will need to provide these permissions:
+    
+    * Create Repositories
+    
+    * View all visible repositories
+    
+    * Read/Write to any accessible repository
+    
+#### Server Configuration
+
+The web service alone only requires a GitHub and Quay.io token. There are two ways to specify your tokens.
+
+1. Environmental variables.
+    You can set them in Bash like the following:
+```
+export quayioToken=<your token here>
+export githubToken=<your token here>
+```
+2. The YAML configuration file. The tokens can be entered in the top two lines. An example of a YAML configuration file can be seen here.
+
+Run the service using a configuration file:
+`java -jar write-api-service-*.jar server example.yml`
+The example.yml shown previously uses port 8082 by default, this can be changed. Note this port number, it will later be used for the Write API Client properties file.
+
+After running the webservice, you can check out the web service endpoints through swagger. By default, it is available at http://localhost:8082/static/swagger-ui/index.html.
+
+The basic workflow is that GitHub repos are created when posting a new tool. When files are posted or put to a version of a tool, we will create or delete and re-create a GitHub release/branch/tag with a matching name. When Dockerfiles are added, the tool will be created and built as a Quay.io repo. After adding both Dockerfiles and descriptors, you basically have a tool that is ready to be quickly registered and published under a Dockstore 1.2 web service. Go to Dockstore, do a refresh, and then hit quick register on the repos that you wish to publish. You can also do this programmatically through the write api client
```

Comment:
Remove "1.2" in Dockstore "1.2" web service. You could say "1.6", but then we'll have to remember it every release.